test/integration_test.go

```diff
@@ -2682,6 +2682,850 @@ func (ts *IntegrationTestSuite) TestUpsertMemoWithExistingMemo() {
+				},
+				{
+					Every: 30 * time.Minute,
+				},
+			},
+			Skip: []client.ScheduleCalendarSpec{
+				{
+					Second: []client.ScheduleRange{{}},
+					Minute: []client.ScheduleRange{{}},
+					Hour: []client.ScheduleRange{{
+						Start: 12,
+					}},
+					DayOfMonth: []client.ScheduleRange{
+						{
+							Start: 1,
+							End:   31,
+						},
+					},
+					Month: []client.ScheduleRange{
+						{
+							Start: 1,
+							End:   12,
+						},
+					},
+					DayOfWeek: []client.ScheduleRange{
+						{
+							Start: 1,
+						},
+					},
+				},
+			},
+		},
+		Action: ts.createBasicScheduleWorkflowAction("test-schedule-describe-spec-workflow"),
+	})
+	ts.NoError(err)
+	ts.EqualValues("test-schedule-describe-spec-schedule", handle.GetID())
+
+	defer func() {
+		ts.NoError(handle.Delete(ctx))
+	}()
+
+	description, err := handle.Describe(ctx)
+	ts.NoError(err)
+	// test spec
+	ts.Equal([]client.ScheduleCalendarSpec{
+		{
+			Second: []client.ScheduleRange{{}},
+			Minute: []client.ScheduleRange{{}},
+			Hour: []client.ScheduleRange{{
+				Start: 12,
+			}},
+			DayOfMonth: []client.ScheduleRange{
+				{
+					Start: 1,
+					End:   31,
+				},
+			},
+			Month: []client.ScheduleRange{
+				{
+					Start: 1,
+					End:   12,
+				},
+			},
+			Year: []client.ScheduleRange{},
+			DayOfWeek: []client.ScheduleRange{
+				{
+					Start: 1,
+				},
+			},
+		},
+	}, description.Schedule.Spec.Calendars)
+
+	ts.Equal([]client.ScheduleIntervalSpec{
+		{
+			Every:  time.Hour,
+			Offset: time.Minute,
+		},
+		{
+			Every: 30 * time.Minute,
+		},
+	}, description.Schedule.Spec.Intervals)
+
+	ts.Equal([]client.ScheduleCalendarSpec{
+		{
+			Second: []client.ScheduleRange{{}},
+			Minute: []client.ScheduleRange{{}},
+			Hour: []client.ScheduleRange{{
+				Start: 12,
+			}},
+			DayOfMonth: []client.ScheduleRange{
+				{
+					Start: 1,
+					End:   31,
+				},
+			},
+			Month: []client.ScheduleRange{
+				{
+					Start: 1,
+					End:   12,
+				},
+			},
+			Year: []client.ScheduleRange{},
+			DayOfWeek: []client.ScheduleRange{
+				{
+					Start: 1,
+				},
+			},
+		},
+	}, description.Schedule.Spec.Skip)
+}
+
+func (ts *IntegrationTestSuite) TestScheduleDescribeSpecCron() {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	handle, err := ts.client.ScheduleClient().Create(ctx, client.ScheduleOptions{
+		ID: "test-schedule-describe-spec-cron-schedule",
+		Spec: client.ScheduleSpec{
+			CronExpressions: []string{
+				"0 12 * * MON",
+			},
+		},
+		Action: ts.createBasicScheduleWorkflowAction("test-schedule-describe-spec-cron-workflow"),
+	})
+	ts.NoError(err)
+	ts.EqualValues("test-schedule-describe-spec-cron-schedule", handle.GetID())
+
+	defer func() {
+		ts.NoError(handle.Delete(ctx))
+	}()
+
+	description, err := handle.Describe(ctx)
+	ts.NoError(err)
+	// test spec
+	ts.Equal([]client.ScheduleCalendarSpec{
+		{
+			Second: []client.ScheduleRange{{}},
+			Minute: []client.ScheduleRange{{}},
```

Comment:
I must be missing something, but here, I would expect an array containing a single proto Range, with `{0, 0, 0}`... Same for `Second`. Isn't that the case?